packages/core/src/components/forms/textArea.tsx

```diff
@@ -170,13 +190,22 @@ export class TextArea extends AbstractPureComponent<TextAreaProps, TextAreaState
             };
         }
 
-        return (
+        return asyncControl ? (
```

Comment:
Does this seem to work when testing live? The code changes look correct. Just wondering if `<TextArea asyncControl />` works as expected.